my_final_project1.py

```diff
@@ -0,0 +1,61 @@
+import time
+print("hello this my final project ")
+ 
+user_name = input("what is your name? ")
+
+print("hi " + user_name + ", nice to meet you")
+
+print("this is a special calculator, i would need two numbers from you")
+
+first_number    = float(input("First Number: "))
+second_number   = float(input("Second Number: "))
+
+print("Thank you for putting in your numbers, " + str(int(first_number)) + " and " + str(int(second_number)))
+
+even_odd1 = first_number % 2 == 0
+if even_odd1:
+    even_odd1 = "even"
+else:
+     even_odd1 = "odd"
+
+print("I can see that the first number is " + even_odd1)
+
+even_odd2 = second_number % 2 == 0
+if even_odd2:
+    even_odd2 = "even"
+else:
+     even_odd2 = "odd"
+
+print("And the second number is " + even_odd2)
+
+if even_odd1 == even_odd2:  
+    if even_odd1 == "odd":
+        both = "odd"
+    elif even_odd1 == "even":
+        both = "even"
+    print("So both of them are " + both)

```

Comment:
אם שניהם שווים - היית יכולה להדפיס
`print("So both of them are " + even_odd1)`
כי גם ככה 1 ו-2 הם אותו דבר, היה חוסך לך את האיף